### DISCUSSION TITLE
How does `list-max-ziplist-size` take effect in Kvrocks?

### DISCUSSION THREAD

**Discussion by PokIsemaine** [apache/kvrocks#2515] [Q&A] [answered]
While browsing ZipList's testcases in gocase, I came across this configuration:
https://github.com/apache/kvrocks/blob/6c85004431d065a6d9c0fd0932c087f9e2e290be/tests/gocase/unit/type/list/list_test.go#L91
It appears to be a Redis configuration.
https://raw.githubusercontent.com/redis/redis/6.2/redis.conf

Starting from Redis 7.0, it changed to `list-max-listpack-size`
https://raw.githubusercontent.com/redis/redis/7.0/redis.conf

I tried searching for `list-max-ziplist-size` within kvrocks but couldn't find where it takes effect, nor is it described in the configuration file.
![image](https://github.com/user-attachments/assets/eb9139ab-0b0d-4429-b109-999d9929e1bb)

I'm wondering where it does take effect.

Thanks!

**Comment by git-hulk** [comment, marked as answer]
@PokIsemaine Yes, you're right. The `list-max-ziplist-size` is meaningless in Kvrocks, can just remove it from the test case.
